llama_stack/providers/inline/agents/meta_reference/openai_responses.py

```diff
@@ -181,10 +231,28 @@ async def create_openai_response(
             chat_response = OpenAIChatCompletion(**chat_response.model_dump())
 
         output_messages: list[OpenAIResponseOutput] = []
-        if chat_response.choices[0].message.tool_calls:
-            output_messages.extend(
-                await self._execute_tool_and_return_final_output(model, stream, chat_response, messages, temperature)
-            )
+        # TODO: should we check more than choices[0] here?
+        if chat_response.choices[0].message.tool_calls and tools:
+            # TODO: Should we support a mix of custom and builtin tools?
```

Comment:
seems OK for now to not support this

Reply:
I agree with regards to mixing custom / builtin tools. I adjusted this comment from a TODO to `# Assume if the first tool is a function, all tools are functions` just so it's clear in case we find a need to support a mix of built-in and custom tools within a single request as users try things out.

Also, it reminded me of the `TODO: should we check more than choices[0] here?` right above there, which I fixed by just iterating over the choices instead of taking `choices[0]`.